fbchat/client.py

```diff
@@ -748,11 +748,12 @@ def fetchThreadMessages(self, thread_id=None, limit=20, before=None):
 
         return list(reversed([graphql_to_message(message) for message in j['message_thread']['messages']['nodes']]))
 
-    def fetchThreadList(self, offset=0, limit=20):
+    def fetchThreadList(self, offset=0, limit=20, type='inbox'):
```

Comment:
Overriding the built in function [type](https://docs.python.org/library/functions.html#type) is bad practice. Perhaps we could rename it to something else?

Reply:
oh this is bad. I'll figure out another good name :)